File: problem_sets/w1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printBlocks(int r);
void printRow(int row);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int row = 0; row < height; row++)
    {
        for (int spaces = 0; spaces < (height - row - 1); spaces++)
        {
            printf(" ");
        }
        printRow(row);
    }
}

void printBlocks(int r)
{
    for (int i = 0; i < (r + 1); i++)
    {
        printf("#");
    }
}

void printRow(int row)
{
    printBlocks(row);
    printf("  ");
    printBlocks(row);
    printf("\n");
}

File: problem_sets/w6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

spaces = 2 * " "

for i in range(1, height + 1):
    alignmentSpaces = (height - i) * " "
    blocks = i * "#"
    print(f"{alignmentSpaces}{blocks}{spaces}{blocks}")
